index.js

```diff
@@ -124,43 +126,59 @@ const drawCommand = (message) => {
           }
           */
 
-          const inventoryRef = userRef.child("inventory");
-          inventoryRef.update({
-            [response.data.id]: {
-              name: response.data.name,
-              image: response.data.card_images[0].image_url,
-              price: response.data.card_prices[0].tcgplayer_price,
-              type: response.data.type,
-              rarity: RARITIES[highestRarityIndex],
-            },
+            const inventoryRef = userRef.child("inventory");
+            inventoryRef.update({
+              [response.data.id]: {
+                name: response.data.name,
+                image: response.data.card_images[0].image_url,
+                price: response.data.card_prices[0].tcgplayer_price,
+                type: response.data.type,
+                rarity: RARITIES[highestRarityIndex],
+              },
+            });
           });
-        });
+      } else {
+        let timeDifferenceInMs = 60000 - timeDifference; //60000 is 1 minute
```

Comment:
i realize this isnt a new code addition, but this and the next line should be `const`